ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-master-presubmits.yaml

```diff
@@ -792,6 +792,149 @@ presubmits:
         secret:
           secretName: regcred
     trigger: (?m)^/test( | .* )e2e-openstack,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    cluster: api.ci
```

Comment:
@alexander-demichev the cluster needs to be `vsphere`

Reply:
See:
https://github.com/openshift/release/blob/master/core-services/sanitize-prow-jobs/_config.yaml#L3217-L3268
Make sure you run `make jobs`